DESCRIPTION

```diff
@@ -17,14 +17,13 @@ URL: https://github.com/tidymodels/probably/,
     https://probably.tidymodels.org
 BugReports: https://github.com/tidymodels/probably/issues
 Depends:
-    R (>= 3.5.0)
+    R (>= 3.5.0), ggplot2
```

Comment:
tune has a bug where `coord_obs_pred()` needs ggplot2 fully loaded to work. I'll fix that in tune and move this back to imports.